main/zoe/guide/contracts/README.md

```diff
@@ -6,27 +6,79 @@ While Zoe provides the means to build custom smart contracts, there is a good ch
 want to use one that follows a commonly-used structure. Therefore, we currently provide several
 pre-built contracts that can be imported and run on Zoe:
 
+## Oracles
+
 - [Oracle Contract](./oracle.md) - the low-level oracle contract that
   can be used for querying [Chainlink
   oracles](https://docs.chain.link/docs/request-and-receive-data#config)
-  or other oracles.
+  or other oracles. 
+
+- [PriceAuthority](/zoe/guide/price-authority.md): To use an price
+  oracle in your own contract, we recommend using the `priceAuthority`
+  higher-level abstraction.
+
+## DeFi Contracts
+
+These contracts create some sort of financial instrument.
+
+
+- [Call Spread](./callSpread.md) - Creates a pair of fully collateralized call spread
+  options. They are ERTP assets and can be used as such in other contracts.
 - [Loan Contract](./loan.md) - a basic collateralized loan contract.
-- [Atomic Swap](./atomic-swap.md) - A basic trade of digital assets between two parties.
+- [Covered Call](./covered-call.md) - Creates a call option, the right
+  to buy an underlying asset.
+- [OTC Desk](./otc-desk.md) - A contract for giving quotes that can be
+  exercised. The quotes are guaranteed to be exercisable because they
+  are actually options with the underlying assets escrowed.
```

Comment:
Now that we're moving the descriptions of the defi contracts together, I want to change my mind about whether to distinguish financially settled vs physically settled. That terminology uses an inappropriate metaphor, so can we say that the call-spread is "financially settled", and the covered call and OTC are "settled in-kind"?

Reply:
That sounds good to me